docs/learn/about-iota/an-introduction-to-iota.md

```diff
@@ -11,61 +11,72 @@ description: A basic introduction to the IOTA DLT Technology. The use-cases and
 
 Blockchains and the Tangle both fall under the same category of distributed ledger technology (DLT).
 
-The two main differences between blockchains and the Tangle are the following:
+The main differences between blockchains and the Tangle are the following:
 
-* - The Tangle has no message fees
-* - IOTA networks have no miners
+- In a Blockchain, new transactions can only be attached to a single point (a new block) that follows and is connected to the previous block. This leads to a race and competition for transactions to be part of a new Block.
+- In the Tangle, every user is free to attach new transactions on different parts of the Tangle. This allows many transactions to be attached and processed in parallel, which dramatically speeds up the processing of transactions
+- A Blockchain must always select a single leader to attach a new Block to the Blockchain. This can lead to the well-known "Miner race," where only the Miner with the highest computational processing power, or Stakers with the highest stake, have a reasonable chance of becoming a Block-Producer and are allowed to attach new Block and to write their transactions to the Ledger. This is also seen as a form of centralization in Blockchains as you need "middlemen" that process your transactions. Regular users are not allowed to write directly to the Ledger
```

Comment:
```suggestion
- A Blockchain must always select a single leader to attach a new Block to the Blockchain. This can lead to the well-known "Miner race," where only the Miner with the highest computational processing power, or Stakers with the highest stake, have a reasonable chance of becoming a Block-Producer and are allowed to attach new Block and to write their transactions to the Ledger. This is also seen as a form of centralization in Blockchains, as you need "middlemen" that process your transactions. Regular users are not allowed to write directly to the Ledger
```